server/log_operation_manager.go

```diff
@@ -185,16 +185,14 @@ func (l *LogOperationManager) masterFor(ctx context.Context, allIDs []int64) ([]
 	if l.info.Registry.ElectionFactory == nil {
 		return allIDs, nil
 	}
-	allIDString := make([]string, 0, len(allIDs))
-	idToString := make(map[int64]string)
+	allStringIDs := make([]string, 0, len(allIDs))
 	for _, id := range allIDs {
 		s := strconv.FormatInt(id, 10)
-		allIDString = append(allIDString, s)
-		idToString[id] = s
+		allStringIDs = append(allStringIDs, s)
```

Comment:
nit: Could just write `allStringIDs[i] = strconv.FormatInt(id, 10)` if `allStringIDs` was created like `make([]string, len(allIDs))`.

Reply:
Indeed, this is the append vs not append argument :-) 
I'm using append to be consistent with the rest of the trillian code base.